hugo/content/exporter/_index.md

```diff
@@ -94,6 +98,15 @@ The following pgMonitor configuration files should be placed according to the fo
 | postgres/queries_*.yml | `/etc/postgres_exporter/##/queries_*.yml`  |
 | postgres/pgbackrest-info.sh | `/usr/bin/pgbackrest-info.sh` |
 
+##### blackbox_exporter
+
+The following pgMonitor configuration files should be placed according to the following mapping:
+
+| pgMonitor Configuration File | System Location |
+|------------------------------|-----------------|
+| blackbox/blackbox_exporter.sysconfig  | `/etc/sysconfig/blackbox_exporter`   |
```

Comment:
```suggestion
| blackbox/blackbox_exporter.sysconfig  | {{ shell }}/etc/sysconfig/blackbox_exporter{{ /shell }}   |
```

Reply:
Actually, I don't think these submodule changes have been applied to the pgmonitor repository. I made sure it was all up to date, but when I run hugo, these formatting flags aren't being applied